docs/DeveloperGuide.md

```diff
@@ -535,27 +530,22 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 **MSS**
 
-1.  User requests to <ins>list flashcards (UC02)</ins>
-2.  User requests to edit the details of a specific flashcard based on the index in the list
-3.  SWEe! edits the flashcard
+1.  User requests to edit the details of a specific flashcard based on the index in the list
+2.  SWEe! edits the flashcard
 
     Use case ends.
 
 **Extensions**
 
-* 1a. The list is empty.
+* 1a. The given index is invalid.
 
-  Use case ends.
-  
-* 2a. The given index is invalid.
-
-    * 2a1. SWEe! shows an error message.
+    * 1a1. SWEe! shows an error message.
 
       Use case ends.
 
-* 2b. The format of the given details are invalid.
+* 1b. The format of the given details are invalid.
```

Comment:
I think can change to the input format is invalid so that it will be consistent?